CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
-0.4.1 (Next)
+0.4.2 (Next)
 ============
 
 * Your contribution here.
 
+0.4.1 (10/23/2015)
```

Comment:
This should be under next, and you can 1-line them, like `Added support for x and y - ...`.